Review: Update German translation for Git v2.34.0

ruester opened the pull request:
@ralfth and @PhillipSz: please have a look at my suggestions for the German translation of Git v2.34.0. Only the commit 88bddee759 needs to be reviewed, the first commit is the automatically generated one to update the po file.

ChrisADR committed:
l10n: es: 2.34.0 round 2

Signed-off-by: Christopher Diaz Riveros <christopher.diaz.riv@gmail.com>
Signed-off-by: Omar Olivares <omar@olivares.cl>
Signed-off-by: Jaime Marquínez Ferrándiz <jaime.marquinez.ferrandiz@fastmail.net>
Signed-off-by: Alex Henrie <alexhenrie24@gmail.com>

robin850 committed:
http-backend: remove a duplicated code branch

Try to make reading the computation of the gzipped flag a bit more
natural.

Signed-off-by: Robin Dupret <robin.dupret@hey.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

raalkml committed:
pull: honor --no-verify and do not call the commit-msg hook

The option was incorrectly auto-translated to "--no-verify-signatures",
which causes the unexpected effect of the hook being called.
And an even more unexpected effect of disabling verification of signatures.

The manual page describes the option to behave same as the similarly
named option of "git merge", which seems to be the original intention
of this option in the "pull" command.

Signed-off-by: Alexander Riesen <raa.lkml@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

raalkml committed:
Document positive variant of commit and merge option "--no-verify"

This documents "--verify" option of the commands. It can be used to re-enable
the hooks disabled by an earlier "--no-verify" in command-line.

Signed-off-by: Alexander Riesen <raa.lkml@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

vdye committed:
async_die_is_recursing: work around GCC v11.x issue on Fedora

This fix corrects an issue found in the `dockerized(pedantic, fedora)` CI
build, first appearing after the introduction of a new version of the Fedora
docker image version. This image includes a version of `glibc` with the
attribute `__attr_access_none` added to `pthread_setspecific` [1], the
implementation of which only exists for GCC 11.X - the version included in
the Fedora image. The attribute requires that the pointer provided in the
second argument of `pthread_getspecific` must, if not NULL, be a pointer to
a valid object. In the usage in `async_die_is_recursing`, `(void *)1` is not
valid, causing the error.

This fix imitates a workaround added in SELinux [2] by using the pointer to
the static `async_die_counter` itself as the second argument to
`pthread_setspecific`. This guaranteed non-NULL, valid pointer matches the
intent of the current usage while not triggering the build error.

[1] https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=a1561c3bbe8
[2] https://lore.kernel.org/all/20211021140519.6593-1-cgzones@googlemail.com/

Co-authored-by: Johannes Schindelin <johannes.schindelin@gmx.de>
Signed-off-by: Victoria Dye <vdye@github.com>
Acked-by: Jeff King <peff@peff.net>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

gitster committed:
Merge branch 'ar/no-verify-doc'

Doc update.

* ar/no-verify-doc:
  Document positive variant of commit and merge option "--no-verify"

gitster committed:
Merge branch 'ar/fix-git-pull-no-verify'

"git pull --no-verify" did not affect the underlying "git merge".

* ar/fix-git-pull-no-verify:
  pull: honor --no-verify and do not call the commit-msg hook

gitster committed:
Merge branch 'rd/http-backend-code-simplification'

Code simplification.

* rd/http-backend-code-simplification:
  http-backend: remove a duplicated code branch

gitster committed:
Merge branch 'vd/pthread-setspecific-g11-fix'

One CI task based on Fedora image noticed a not-quite-kosher
consturct recently, which has been corrected.

* vd/pthread-setspecific-g11-fix:
  async_die_is_recursing: work around GCC v11.x issue on Fedora

gitster committed:
Merge branch 'jk/log-warn-on-bogus-encoding' into maint

Squelch over-eager warning message added during this cycle.

* jk/log-warn-on-bogus-encoding:
  log: document --encoding behavior on iconv() failure
  Revert "logmsg_reencode(): warn when iconv() fails"

gitster committed:
Merge branch 'pw/rebase-r-fixes' into maint

Regression fix.

* pw/rebase-r-fixes:
  rebase -i: fix rewording with --committer-date-is-author-date

gitster committed:
Merge branch 'ma/doc-git-version' into maint

Typofix.

* ma/doc-git-version:
  git.txt: fix typo

gitster committed:
Merge branch 'vd/pthread-setspecific-g11-fix' into maint

One CI task based on Fedora image noticed a not-quite-kosher
consturct recently, which has been corrected.

* vd/pthread-setspecific-g11-fix:
  async_die_is_recursing: work around GCC v11.x issue on Fedora

gitster committed:
Sync with maint

gitster committed:
A few fixes before -rc2

Signed-off-by: Junio C Hamano <gitster@pobox.com>

me-and committed:
t/lib-git.sh: fix ACL-related permissions failure

As well as checking that the relevant functionality is available, the
GPGSSH prerequisite check creates the SSH keys that are used by the test
functions it gates.  If these keys are created in a directory that
has a default Access Control List, the key files can inherit those
permissions.

This can result in a scenario where the private keys are created
successfully, so the prerequisite check passes and the tests are run,
but the key files have permissions that are too permissive, meaning
OpenSSH will refuse to load them and the tests will fail.

To avoid this happening, before creating the keys, clear any default ACL
set on the directory that will contain them.  This step allowed to fail;
if setfacl isn't present, that's a very likely indicator that the
filesystem in question simply doesn't support default ACLs.

Helped-by: Fabian Stelzer <fs@gigacodes.de>
Signed-off-by: Adam Dinwoodie <adam@dinwoodie.org>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

alshopov committed:
l10n: bg.po: Updated Bulgarian translation (5210t)

Signed-off-by: Alexander Shopov <ash@kambanaria.org>

ruester committed:
Update de.po for new translations

Signed-off-by: Matthias Rüster <matthias.ruester@gmail.com>

ruester committed:
l10n: de.po: Update German translation

Signed-off-by: Matthias Rüster <matthias.ruester@gmail.com>

vnwildman committed:
l10n: vi(5210t): Translation for v2.34.0 rd2

Signed-off-by: Tran Ngoc Quan <vnwildman@gmail.com>

PhillipSz commented on po/de.po:
```diff
@@ -11309,14 +11320,12 @@ msgid "git bisect--helper --bisect-skip [(<rev>|<range>)...]"
-#, fuzzy
 msgid "git bisect--helper --bisect-run <cmd>..."
-msgstr "git bisect--helper --bisect-reset [<Commit>]"
+msgstr "git bisect--helper --bisect-run <Programm>"
```
würde das "…" behalten

ruester replied:
Vielen Dank! :smiley:

PhillipSz reviewed (changes requested):
Hallo Matthias,

vielen Dank für deine Übersetzungen. Habe nur eine Sache gefunden, sonst passt alles!

Viele Grüße
Phllip

ralfth commented on po/de.po:
```diff
@@ -4389,71 +4396,81 @@ msgid ""
 #: gpg-interface.c:895
 msgid "user.signingkey needs to be set for ssh signing"
-msgstr ""
+msgstr "user.signingkey muss für die SSH-Signatur festgelegt werden"
```
... gesetzt sein
?

ruester replied:
ja, besser, danke :smiley:

ralfth commented on po/de.po:
```diff
@@ -4604,7 +4621,7 @@ msgstr "Setze fort unter der Annahme, dass Sie '%s' meinten."
 #, c-format
 msgid "Run '%s' instead? (y/N)"
-msgstr ""
+msgstr "Führen Sie stattdessen '%s' aus? (y/N)"
```
Stattdessen '%s' ausführen? (y/N)

ruester replied:
ja, viel besser :+1:

ralfth commented on po/de.po:
```diff
@@ -5490,27 +5507,26 @@ msgid "cannot store reverse index file"
+#, c-format
 msgid "could not parse line: %s"
-msgstr "konnte %s nicht parsen"
+msgstr "Zeile konnte nicht analysiert werden: %s"
```
...geparst werden...
Das verwenden wir schon an anderen Stellen.

ralfth commented on po/de.po:
```diff
@@ -18764,18 +18761,16 @@ msgstr ""
-#, fuzzy
 msgid "write multi-pack index containing only given indexes"
-msgstr "git multi-pack-index [<Optionen>] verify"
+msgstr "schreibe nur Multipack-Bitmap-Index, die vorgegebene Indexe enthalten"
```
Multipack-Index schreiben, der nur die gegebenen Indexe enthält

ruester replied:
ohje :see_no_evil:  danke!

ralfth commented on po/de.po:
```diff
@@ -23631,62 +23616,60 @@ msgid "Failed to clone '%s' a second time, aborting"
+#, c-format
 msgid "Unable to merge '%s' in submodule path '%s'"
-msgstr "Merge von '$sha1' in Submodul-Pfad '$displaypath' fehlgeschlagen"
+msgstr "Merge von '%s' in Submodul-Pfad '%s' fehlgeschlagen"
```
... nicht möglich

Wäre dann dieselbe Übersetzung wie oben.

ralfth commented on po/de.po:
```diff
@@ -23904,55 +23884,53 @@ msgid "unable to checkout submodule '%s'"
+#, c-format
 msgid "'%s' already exists in the index"
-msgstr "'$sm_path' ist bereits zum Commit vorgemerkt"
+msgstr "'%s' ist bereits zum Commit vorgemerkt"
```
Ich denke,
'%s' existiert bereits im Index
ist hier besser.

Das war wohl schon vorher falsch.

ralfth commented on po/de.po:
```diff
@@ -23904,55 +23884,53 @@ msgid "unable to checkout submodule '%s'"
+#, c-format
 msgid "'%s' already exists in the index and is not a submodule"
-msgstr "'$sm_path' ist bereits zum Commit vorgemerkt und ist kein Submodul"
+msgstr "'%s' ist bereits zum Commit vorgemerkt und ist kein Submodul"
```
dito
... existiert bereits im Index...

ralfth reviewed (changes requested):
Danke für die Übersetzungen! Gute Arbeit! Nur ein paar kleine Anmerkungen.

avar committed:
parse-options.[ch]: revert use of "enum" for parse_options()

Revert the parse_options() prototype change in my recent
352e761388b (parse-options.[ch]: consistently use "enum
parse_opt_result", 2021-10-08) was incorrect. The parse_options()
function returns the number of argc elements that haven't been
processed, not "enum parse_opt_result".

Reported-by: SZEDER Gábor <szeder.dev@gmail.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>

Arusekk committed:
l10n: pl: 2.34.0 round 2

Signed-off-by: Arusekk <arek_koz@o2.pl>

gitster committed:
Merge branch 'ab/parse-options-cleanup'

Last minute fix to the update already in 'master'.

* ab/parse-options-cleanup:
  parse-options.[ch]: revert use of "enum" for parse_options()

gitster committed:
Merge branch 'ad/ssh-signing-testfix'

Fix ssh-signing test to work on a platform where the default ACL is
overly loose to upset OpenSSH (reported on an installation of Cygwin).

* ad/ssh-signing-testfix:
  t/lib-git.sh: fix ACL-related permissions failure

gitster committed:
Git 2.34-rc2

Signed-off-by: Junio C Hamano <gitster@pobox.com>

jiangxin committed:
Merge branch 'master' of github.com:git/git

* 'master' of github.com:git/git:
  Git 2.34-rc2
  parse-options.[ch]: revert use of "enum" for parse_options()
  t/lib-git.sh: fix ACL-related permissions failure
  A few fixes before -rc2
  async_die_is_recursing: work around GCC v11.x issue on Fedora
  Document positive variant of commit and merge option "--no-verify"
  pull: honor --no-verify and do not call the commit-msg hook
  http-backend: remove a duplicated code branch

jiangxin committed:
l10n: git.pot: v2.34.0 round 3 (1 new)

Generate po/git.pot from v2.34.0-rc2 for git v2.34.0 l10n round 3.

Signed-off-by: Jiang Xin <worldhello.net@gmail.com>

jiangxin committed:
Merge branch 'master' of github.com:alshopov/git-po

* 'master' of github.com:alshopov/git-po:
  l10n: bg.po: Updated Bulgarian translation (5210t)

fangyi-zhou committed:
l10n: zh_CN: v2.34.0 round 3

Signed-off-by: Fangyi Zhou <me@fangyi.io>

bitigchi committed:
l10n: tr: v2.34.0 round 3

Signed-off-by: Emir Sarı <bitigchi@me.com>

jnavila committed:
l10n: fr: v2.34.0 rnd 3

Signed-off-by: Jean-Noël Avila <jn.avila@free.fr>

pan93412 committed:
l10n: zh_TW.po: v2.34.0 round 3 (0 untranslated)

Signed-off-by: Yi-Jyun Pan <pan93412@gmail.com>

jiangxin committed:
Merge branch 'tr-2-34-r3' of github.com:bitigchi/git-po

* 'tr-2-34-r3' of github.com:bitigchi/git-po:
  l10n: tr: v2.34.0 round 3

jiangxin committed:
Merge branch 'fr_v2.34.0_rnd3' of github.com:jnavila/git

* 'fr_v2.34.0_rnd3' of github.com:jnavila/git:
  l10n: fr: v2.34.0 rnd 3

jiangxin committed:
Merge branch 'l10n/zh_TW/211111' of github.com:l10n-tw/git-po

* 'l10n/zh_TW/211111' of github.com:l10n-tw/git-po:
  l10n: zh_TW.po: v2.34.0 round 3 (0 untranslated)

vnwildman committed:
l10n: vi(5211t): Translation for v2.34.0 rd3

Signed-off-by: Tran Ngoc Quan <vnwildman@gmail.com>

nafmo committed:
l10n: sv.po: Update Swedish translation (5211t0f0)

Signed-off-by: Peter Krefting <peter@softwolves.pp.se>

ruester committed:
Make translations containing '...' to look always the same

ruester committed:
Apply suggestions from Ralf

ruester commented:
Vielen Dank für euer Review, @PhillipSz und @ralfth !
Habe ich alles so übernommen :wink: :+1: 
Schicke ich dann jetzt als Pull Request zum Maintainer...

ruester committed:
Fix placeholder value

ruester committed:
Update de.po for new translations (round #3)

ruester committed:
Add translation from round #3

ruester commented:
I had to fix one more message, see 6b3208aeb2fe35c0a8332ee7039bfb329a5e7edf
Also one new message was added in translation round #⁠3, see f36265d1e9e675ef85a880eedb377a9a7b254f8e

ruester commented:
https://github.com/git-l10n/git-po/pull/594
